src/EduardTrayan/CommonBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,3 @@
+eduard_trayan_common_homepage:
```

Comment:
Please do the same thing as well as in `EduardTrayan/WebsiteBundle/Resources/config/routing.yml`